Sources/MapboxNavigation/MapView.swift

```diff
@@ -50,24 +61,24 @@ extension MapView {
      - parameter tileSetIdentifier: Identifier of the tile set in the form `user.tileset`.
      - returns: Set of source identifiers.
      */
-    func sourceIdentifiers(_ tileSetIdentifier: String) -> Set<String> {
+    func sourceIdentifiers(_ tileSetIdentifiers: Set<String>) -> Set<String> {
         return Set(mapboxMap.style.allSourceIdentifiers.filter {
             $0.type.rawValue == "vector"
         }.filter {
-            tileSetIdentifiers($0.id).contains(tileSetIdentifier)
+            !tileSetIdentifiersSet($0.id).isDisjoint(with: tileSetIdentifiers)
         }.map {
             $0.id
         })
     }
     
     /**
-     Returns a Boolean value indicating whether data from the given tile set layer is currently visible in the map view’s style.
+     Returns a Boolean value indicating whether data from the given tile set layers is currently visible in the map view’s style.
      
-     - parameter tileSetIdentifier: Identifier of the tile set in the form `user.tileset`.
+     - parameter tileSetIdentifiers: Identifiers of the tile sets in the form `user.tileset`.
      - parameter layerIdentifier: Identifier of the layer in the tile set; in other words, a source layer identifier. Not to be confused with a style layer.
      */
-    func showsTileSet(withIdentifier tileSetIdentifier: String, layerIdentifier: String) -> Bool {
-        let sourceIdentifiers = self.sourceIdentifiers(tileSetIdentifier)
+    public func showsTileSet(withIdentifiers tileSetIdentifiers: Set<String>, layerIdentifier: String) -> Bool {
```

Comment:
Does this function work as expected? Do we want it to return `true` only in case if one of the tile set identifiers is currently shown or all of them? Based on documentation it looks unclear.

Reply:
Yes. Because when we have a set of tile sets, we may want to check whether all of them are currently shown. Thanks for the suggestion, I'll improve this part.